packages/itwinui-css/src/tabs/tabs.scss

```diff
@@ -136,6 +161,12 @@
       width: var(--iui-size-3xs);
     }
   }
+
+  .iui-tabs-actions {
+    flex-direction: column;
```

Comment:
i think these should be left aligned when vertically stacked. center alignment looks weird
![](https://user-images.githubusercontent.com/9084735/210437580-dba2b3f6-2a86-4288-b0cd-37b18efda4a3.png)

Reply:
This is how it looks with left alignment. Which one looks better @mayank99 @FlyersPh9 ? Because I feel the center one looks better. The left aligned one seems a bit off because it is misaligned with the `+` icon in the bottom most tab that is center aligned.

![image](https://user-images.githubusercontent.com/45748283/210440612-8895bb49-46de-42d3-8c91-528ccac0c26e.png)

Reply:
we might want to make those buttons take the full width then

Reply:
Does this look alright? Apart from `align-items: stretch`, I also added `padding: 0 var(--iui-size-xs)`.

![image](https://user-images.githubusercontent.com/45748283/210594912-0323cc51-64fd-423a-b775-bc4518c99cdc.png)

Reply:
I like the full width option.  Any negative impact when using components other than buttons?  Checkbox with a label comes to mind.

Reply:
I tried with a few components other than buttons. I noticed issues in `tag`, `keyboard`, and `badge`.

But I believe this is expected behavior because this happens even without `align-items: stretch`. So the below screenshot happens with or without `align-items: stretch`.

![image](https://user-images.githubusercontent.com/45748283/210853834-1b876432-8086-4fdf-874b-6d6cea906a52.png)